monero-outreach-docs/translations/fr/merchants/merchant_FAQ_fr.md

```diff
@@ -0,0 +1,57 @@
+
+### _Les transactions avec Monero sont-elles moins dispendieuses qu'avec Bitcoin?_
+
+Aussi complexe que les transactions par crédit/débit peuvent être, les frais tendent à être réguliers. Le Frais d'Interchange est régi par la loi et mis à jour deux fois par année. En comparaison, les frais d'utilisation de Bitcoin sont erratiques, irréguliers, et augmentent lorsque le réseau est sollicité. Les frais de transaction pour Monero, quant à eux, sont demeurés bas et stables depuis l'introduction de Bulletproofs en 2018.
+
+Les frais d'utilisation de Bitcoin varient constamment, et ce tableau permet d'en suivre l'évolution : [bitinfocharts.com/comparison...fees-btc-xmr...](https://bitinfocharts.com/comparison/transactionfees-btc-xmr-sma7.html#1y). Pour avoir une meilleure idée du ratio des frais BTC/XMR à un moment donné, [monero.how/monero-transaction-fees](https://www.monero.how/monero-transaction-fees) suit les hauts, les bas, et les moyennes pour les 100 dernières transactions sur chaque réseau.
+
+### _Y a-t-il une manière facile et automatique de convertir les XMR du client en monnaie fiduciaire?_
+
+La plupart des fournisseurs de services de paiement offrent le choix de la monnaie dans laquelle vos transactions seront finalisées, incluant la monnaie fiduciaire. Notre [Guide de Fournisseurs de Services de Paiement Monero](https://www.monerooutreach.org/merchants/monero-payment-processor-guide.html) offre plus de détails. Si vous utilisez un service de paiement auto-hébergé, ou si vous acceptez Monero directement dans un portefeuille, alors vous devrez vous connecter à l'API d'un échange pour effectuer la conversion automatiquement.
+
+### _Comment déclare-t-on ces revenus pour des fins d'impôt?_
+
+Vous devrez chercher conseil auprès de votre comptable ou conseiller en impôt. La plupart du temps, l'achat est calculé selon la monnaie locale pour que l'ensemble de votre comptabilité soit cohérente et que les taxes à l'achat soient adéquatement perçues. Ce total serait ensuite converti en XMR selon le taux courant. Cette méthode de procéder est plutôt courante dans les villes frontalières qui doivent fréquemment accepter plus qu'une monnaie. Les transactions sont d'abord enregistrées dans votre monnaie locale, et la conversion est appliquée par après. Lorsque vous avez été payé en XMR et le client a pris ses biens, vous auriez seulement l'évènement imposable normal si vous dépensiez ce même XMR ou l'échangiez pour une autre monnaie. Ce second évènement imposable n'est pas unique aux transactions de cryptomonnaies, et même une transaction par carte de crédit peut créer un évènement imposable additionnel tout dépendant de la manière dont vous disposez des fonds.
```

Comment:
1.
la conversion est appliquée par après  >>>   la conversion est appliquée a posteriori   

Reason: Never saw 'par après' in a sentence. 'a posteriori' sound better to me

2.
Lorsque vous avez été payé en XMR et le client a pris ses biens  >>>   Lorsque vous avez été payé en XMR et que le client a pris ses biens     

Reason: Missing conjonction

3.
Original sentence:
un évènement imposable additionnel tout dépendant de la manière dont vous disposez des fonds
Suggested sentence:
un évènement imposable additionnel qui dépendra de la manière dont vous disposez des fonds   

Reason: Alternative translation which sounds better to me

Reply:
Recommendations were integrated.